### PR TITLE
Release v1.7.0

### PR DESCRIPTION
**Pull request by yuya-takeyama**
@woothee/owners 

Sorry, I did some changes at once.

## Parser update

It contains changes of `1.6.0` and `1.7.0`.

* `1.6.0`
  * curl https://github.com/woothee/woothee/pull/30
  * WebView on Android https://github.com/woothee/woothee/pull/33
* `1.7.0`
  * Yeti/1.1 https://github.com/woothee/woothee/pull/36
  * trendictionbot https://github.com/woothee/woothee/pull/37

## Other changes

I added Ruby 2.3.3 and 2.4.1 to `.travis.yml`. https://github.com/woothee/woothee-ruby/commit/fae20c6d4600d7d9583a3514e30bac62d4d60631

Additionally, I removed `2.0` and `2.1` since their maintenance statuses are already EOL. https://github.com/woothee/woothee-ruby/commit/25bbda88e3eb34fde1f6d758ed278355f0962fdd
ref: https://www.ruby-lang.org/en/downloads/branches/

### CODE REVIEW

File: lib/woothee/crawler.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@ def self.challenge_crawlers(ua, result)
       end
     end
     if ua.index('Yeti')
-      if ua.index('http://help.naver.com/robots')
+      if ['http://help.naver.com/robots', 'http://help.naver.com/support/robots.html', 'http://naver.me/bot'].any? { |url| ua.index(url) }
```
Comment (tagomoris) [inline lib/woothee/crawler.rb]: This code creates Array object every time.
Use just 3 `ua.index(uri)` with `||`, or a constant of 3 URIs.

Comment (yuya-takeyama) [inline lib/woothee/crawler.rb]: @tagomoris 
Ah, that's true...
Fixed it.
https://github.com/woothee/woothee-ruby/pull/15/commits/6e3424722cd08117214ce9d389db6cc0a69c3652

**Comment by tagomoris** [comment]
@yuya-takeyama Could you commit to bump version number after the last code change?

**Comment by yuya-takeyama** [comment]
@tagomoris 
You mean about the order of the commits?
Did it.

**Comment by tagomoris** [comment]
Thank you!
LGTM.

**Pull request merged**